datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/agent_signature.py

```diff
@@ -29,8 +33,15 @@ def legacy_signature(check):
         check_failed = find_legacy_signature(check)
         if check_failed is not None:
             has_failed = True
-            failed_file, failed_num = check_failed
+            failed_file_path, failed_num = check_failed
+            failed_file = os.path.basename(failed_file_path)
             echo_failure(f"Check `{check}` uses legacy agent signature in `{failed_file}` on line {failed_num}")
+            print_github_annotation(
+                failed_file_path,
+                "Detected use of legacy agent signature, please use the new signature",
+                level="error",
+                line=failed_num + 1,
```

Comment:
Good catch, do you mind updating it in `find_legacy_signature` so the line number is printed correctly too?